### PR TITLE
[DO NOT MERGE] Prepopulate extraction GUI

### PR DESCRIPTION
**Pull request by josenavas**
~~Depends on #100 so merge that one first.~~

Now the user can go back to the extraction page and the GUI will prepopulate for the existing process.

Example:

extract

### CODE REVIEW

**Comment by josenavas** [comment]
This is ready for a review - @tanaes, can you?

File: labman/gui/handlers/process_handlers/gdna_extraction_process.py [hunk 1]
```diff
@@ -22,12 +22,26 @@ class GDNAExtractionProcessHandler(BaseHandler):
     @authenticated
     def get(self):
         plate_ids = self.get_arguments('plate_id')
+        process_id = self.get_argument('process_id', None)
+        plates_info = {}
+        if process_id:
```
Comment (wasade) [inline labman/gui/handlers/process_handlers/gdna_extraction_process.py]: `is not None`

**Comment by josenavas** [comment]
I think this is ready for merge

**Comment by josenavas** [comment]
Closing in favor of #124, since these changes have been rolled in there given that I had to modify the interface.

**Pull request closed without merging**